components/collapsible-panel/collapsible-panel.js

```diff
@@ -97,6 +97,7 @@ class CollapsiblePanel extends RtlMixin(LitElement) {
 				--d2l-collapsible-panel-spacing-inline: 2rem;
 			}
 			.d2l-collapsible-panel {
+				background-color: #ffffff;
```

Comment:
I've seen cases where the background colour behind the panel bleeds through:

![image](https://user-images.githubusercontent.com/5491151/219386341-c22d0376-66a1-4f15-805e-0529c25e8eab.png)

This would likely impact those? [Slack thread](https://d2l.slack.com/archives/C03NDJRSE1Y/p1675710396572409).

Reply:
The background-color case is a bit more complicated and deserves its own PR so I am going to remove it from here. I'll open a separate PR to address it.